packages/tasks/src/model-libraries.ts

```diff
@@ -946,6 +946,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: false,
 		countDownloads: `path:"vae_stats.json"`,
 	},
+	"tencent-song-generation": {
+		prettyLabel: "SongGeneration",
+		repoName: "songgeneration",
+		repoUrl: "https://github.com/tencent-ailab/songgeneration",
+		filter: false,
+		countDownloads: `path:"ckpt/songgeneration_base_zh/model.pt"`,
```

Comment:
I think there is a typo in the path. The rule must be updated if we want to count download using https://huggingface.co/tencent/SongGeneration/blob/main/ckpt/songgeneration_base/model.pt 

```suggestion
		countDownloads: `path:"ckpt/songgeneration_base/model.pt"`,
```

I opened https://github.com/huggingface/huggingface.js/pull/1554 to fix it